core/include/cubos/core/al/audio_context.hpp

```diff
@@ -65,13 +69,17 @@ namespace cubos::core::al
         /// @param listenerCount Number of audio listeners to be supported by the device.
         /// @param specifier Identifier of the audio device.
         /// @return Handle of the new device
-        virtual AudioDevice createDevice(unsigned int listenerCount, const std::string& specifier = "") = 0;
+        virtual AudioDevice createDevice(unsigned int listenerCount) = 0;
```

Comment:
Why is it no longer possible to choose a specific audio device?

Reply:
I forgot to revert this when debugging the sample, I'll add it back again!